LICENSE

```diff
@@ -1,21 +1,14 @@
-MIT License
+BSD 3-Clause License
 
-Copyright (c) 2020 Michael Hoefer
```

Comment:
I noticed we were using a really old license that has Michael Hoefer's name on it. I switched us to use the official BSD-3-Clause license that salesforce open source projects are supposed to have.

Reply:
Are there rules about license changes? Can we just change the license like that?